distributed/protocol/tests/test_netcdf4.py

```diff
@@ -1,10 +1,13 @@
 import pytest
 
-netCDF4 = pytest.importorskip("netCDF4")
-np = pytest.importorskip("numpy")
+import dask.array as da
 
 from distributed.protocol import deserialize, serialize
 from distributed.utils import tmpfile
+from distributed.utils_test import gen_cluster
+
+netCDF4 = pytest.importorskip("netCDF4")
+np = pytest.importorskip("numpy")
```

Comment:
This skip needs to be before `import dask.array as da` (or we need to add a skip for `dask.array`)